patroni/api.py

```diff
@@ -377,8 +381,10 @@ def get_postgresql_status(self, retry=False):
                                        pg_xlog_location_diff(pg_last_xlog_receive_location(), '0/0')::bigint,
                                        pg_xlog_location_diff(pg_last_xlog_replay_location(), '0/0')::bigint,
                                        to_char(pg_last_xact_replay_timestamp(), 'YYYY-MM-DD HH24:MI:SS.MS TZ'),
-                                       pg_is_in_recovery() AND pg_is_xlog_replay_paused()""", retry=retry)[0]
-            return {
+                                       pg_is_in_recovery() AND pg_is_xlog_replay_paused(),
+                                       (SELECT json_agg(row_to_json(ri)) FROM replication_info ri)""", retry=retry)[0]
```

Comment:
Hey guys!
A quick question - what is actually currently the min. supported Pg version for Patroni? I could  I think these JSON functions came only in 9.3. Would be nice to cite it also on the main README. Cheers